File: social_platform/app/core/content_limits.py
"""公开社交内容长度约束。

该模块集中定义数据库模型和 API schema 共用的正文上限，避免不同写入入口产生偏差。
"""

ARTICLE_CONTENT_MAX_LENGTH = 10_000
"""文章正文允许的最大字符数。"""

POST_CONTENT_MAX_LENGTH = 1_000
"""普通帖子（包括转发附言）允许的最大字符数。"""

COMMENT_CONTENT_MAX_LENGTH = 1_000
"""评论和回复允许的最大字符数。"""

File: social_platform/frontend/src/shared/config/contentLimits.ts
/**
 * 公开内容输入上限。
 *
 * 这些值与公开平台数据库约束和 Pydantic schema 保持一致，供所有发布入口复用。
 */
export const ARTICLE_CONTENT_MAX_LENGTH = 10_000;
export const POST_CONTENT_MAX_LENGTH = 1_000;
export const COMMENT_CONTENT_MAX_LENGTH = 1_000;
